app/controllers/application_controller.rb

```diff
@@ -1,5 +1,5 @@
+require 'will_paginate/array'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.